File: nomad-samples/admin/py/basic-live-channel-1.1/schedule_event/remove_input_schedule_event.py
from constants.project_constants import *
from exceptions.api_exception_handler import *

import requests

def remove_input_schedule_event(AUTH_TOKEN, CHANNEL_ID, INPUT_ID):

    API_URL = f"{ADMIN_URL}/liveChannel/{CHANNEL_ID}/liveScheduleEvent/{INPUT_ID}" 

    # Create header for the request
    HEADERS = {
        'Content-Type': 'application/json',
        "Authorization": "Bearer " + AUTH_TOKEN
    }

    try:
        RESPONSE = requests.delete(API_URL, headers= HEADERS)

        if not RESPONSE.ok:
            raise Exception()
        
    except:
        api_exception_handler(RESPONSE, "Remove Input Schedule Event failed")

File: nomad-samples/admin/js/basic-live-channel/schedule-event/remove-input-schedule-event.js
import * as prjConstants from "../constants/project-constants.js";
import apiExceptionHandler from "../exceptions/api-exception-handler.js";

export default async function removeInputScheduleEvent(AUTH_TOKEN, CHANNEL_ID, INPUT_ID)
{
    // Create header for the request
    const HEADERS = new Headers();
    HEADERS.append("Content-Type", "application/json");
    HEADERS.append("Authorization", `Bearer ${AUTH_TOKEN}`);


    // Post
    const RESPONSE = await fetch(`${prjConstants.PORTAL_API_URL}/liveChannel/${CHANNEL_ID}/liveScheduleEvent/${INPUT_ID}`, {
        method: "delete",
        headers: HEADERS,
        body: JSON.stringify(BODY)
    }).catch((exception) => {
        throw exception;
    });

    // Check for success
    if (RESPONSE.ok) {
        
    }

    // There was an error
    await apiExceptionHandler(RESPONSE, "Failed to remove input schedule event");
}